.circleci/config.yml

```diff
@@ -250,14 +250,13 @@ jobs:
 
   macos-integration-test:
     description: "Run integration tests for Flutter"
-    resource_class: macos.x86.medium.gen2
+    resource_class: macos.m1.large.gen1
```

Comment:
Hmm I believe these machines are much more expensive though... What are we using in the `purchases-ios` repo?

Reply:
oh wow yes, I just checked and they are 400 credits vs 50... Let's wait until they release a more recent version of Ventura with their x86 machine then. I think that will fix it like it did in GitHub Actions.

Thanks for pointing this out!